src/unix/bsd/netbsdlike/openbsdlike/openbsd/mod.rs

```diff
@@ -25,8 +25,219 @@ s! {
         pub int_p_sign_posn: ::c_char,
         pub int_n_sign_posn: ::c_char,
     }
+
+    pub struct statfs {
+        pub f_flags: ::uint32_t,
+        pub f_bsize: ::uint32_t,
+        pub f_iosize: ::uint32_t,
+        pub f_blocks: ::uint64_t,
+        pub f_bfree: ::uint64_t,
+        pub f_bavail: ::int64_t,
+        pub f_files: ::uint64_t,
+        pub f_ffree: ::uint64_t,
+        pub f_favail: ::int64_t,
+        pub f_syncwrites: ::uint64_t,
+        pub f_syncreads: ::uint64_t,
+        pub f_asyncwrites: ::uint64_t,
+        pub f_asyncreads: ::uint64_t,
+        pub f_fsid: ::fsid_t,
+        pub f_namemax: ::uint32_t,
+        pub f_owner: ::uid_t,
+        pub f_ctime: ::uint64_t,
+        pub f_fstypename: [::c_char; 16],
+        pub f_mntonname: [::c_char; 90],
+        pub f_mntfromname: [::c_char; 90],
+        pub f_mntfromspec: [::c_char; 90],
+        pub mount_info: mount_info,
+    }
+
+    pub union mount_info {
```

Comment:
I have the following error will trying to run testsuite on openbsd:
```
error: no rules expected the token `union`
  --> src/unix/bsd/netbsdlike/openbsdlike/openbsd/mod.rs:54:9
   |
54 |     pub union mount_info {
   |         ^^^^^
```

The block is in `s!` marco block, and `union` seems unsupported by the macro

Reply:
I fixed unions. Can you rerun the testsuite?